collab2/foraging/toolkit/visualization.py

```diff
@@ -0,0 +1,74 @@
+from typing import List
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+
+
+def plot_predictor(
```

Comment:
I think it is unclear what the intended behavior of some of these things is, variable names like f, t are meaningless, and it is not obvious to me whether this will be sufficiently general to handle an arbitrary predictor.

Reply:
Agreed. Changed `f`,`t` to `forager`,`time` so it is more clear. Added a docstring comment to explain function parameters. 
I'm not sure why you think that the function is not generalizable, and it is hard for me to clarify your doubts without more information. The function accepts any predictor (in the format of nested lists that is returned by the corresponding `generate_predictor` functions) and allows us to inspect the computed predictor values by visualizes it for a chosen set of foragers and time frames.